Sources_v3/Controllers/ChannelController/ChannelController.swift

```diff
@@ -695,12 +750,22 @@ public extension _ChatChannelController where ExtraData == DefaultExtraData {
     }
 }
 
+/// Describes the flow of the items in the list
+public enum ListOrdering {
```

Comment:
I thought we went with `MessageFlow` for this :/

Reply:
We agreed on `ListDirection` ([slack message](https://getstream.slack.com/archives/CRML30N0H/p1599058682028300?thread_ts=1599055708.017100&cid=CRML30N0H)), I'm not sure why it's still `ListOrdering` 🤷‍♂️